app/models/flood_risk_engine/enrollment_exemption.rb

```diff
@@ -20,5 +20,21 @@ def self.include_long_dredging?
       includes(:exemption).where(flood_risk_engine_exemptions: { code: Exemption::LONG_DREDGING_CODES }).any?
     end
 
+    def decision_at_and_user
+      return [nil, nil] if comments.empty?
+      latest = comments.order(:created_at).last
+      [latest.created_at, User.find(latest.user_id).try(:email)]
+    end
+
+    def decision_at
+      return if comments.empty?
+      comments.order(:created_at).last.created_at
+    end
+
+    def decision_user_id
+      return if comments.empty?
+      comments.order(:created_at).last.user_id
+    end
+
```

Comment:
I'd suggest adding a latest comment method that could then be used in all three methods:
```ruby
def latest_comment
  comments.order(:created_at).last
end
```